src/Resources/public/js/main.js

```diff
@@ -170,7 +170,7 @@ document.addEventListener('DOMContentLoaded', function() {
 		}
 		return data.labels[key];
 	};
-	var switchBackend = function(url, requestType, data, callback) {
+	var postToIframe = function(url, data, isXmlHttpRequest, callback) {
```

Comment:
If you do not want to rename that method to prevent BC break, the new parameter should be added last, shouldn't it?

Reply:
Renamed in abd7a6a97858ee9980ba9a19790089e10b711673.

The method is private so there should be no BC break I think.